explorer/interpreter/action_stack.cpp

```diff
@@ -192,32 +204,64 @@ auto ActionStack::RunAgain() -> ErrorOr<Success> {
   return Success();
 }
 
-auto ActionStack::UnwindTo(Nonnull<const Statement*> ast_node)
-    -> ErrorOr<Success> {
+auto ActionStack::UnwindToWithCaptureScopesToDestroy(
+    Nonnull<const Statement*> ast_node) -> std::list<std::unique_ptr<Action>> {
```

Comment:
Would it work to use `std::stack` instead of `std::list` to track the actions to destroy? I think that could be clearer about the order that the actions will be cleaned up, and it would be more efficient.

Reply:
Done!